system/Test/Mock/MockCache.php

```diff
@@ -31,7 +32,12 @@ class MockCache extends BaseHandler implements CacheInterface
 	 */
 	protected $expirations = [];
 
-	//--------------------------------------------------------------------
+	/**
+	 * If true, will not cache any data.
+	 *
+	 * @var boolean
+	 */
+	protected $bypass = false;
```

Comment:
If the `MockCache` class is already being used by the developer, or through the `mock()` helper, there is no way to specify the `dummy` handler. I think if someone wanted to use that they would have to set that in `phpunit.xml` for testing environment. But if you are using the `MockCache` class it will always act as a cache engine. If you want to bypass that and ensure that the cache is a miss, you could use `bypass`.